src/hotspot/share/gc/g1/g1BlockOffsetTable.inline.hpp

```diff
@@ -46,7 +46,7 @@ inline HeapWord* G1BlockOffsetTablePart::block_start_reaching_into_card(const vo
 
   size_t index = _bot->index_for(addr);
 
-  uint offset = _bot->offset_array(index);
+  u_char offset = _bot->offset_array(index);
```

Comment:
pre-existing: the offset array base type should probably be `uint8_t`, not `u_char`, but that's a cleanup for another CR.